mopidy_mpd/protocol/connection.py

```diff
@@ -51,3 +52,54 @@ def ping(context):
         Does nothing but return ``OK``.
     """
     pass
+
+
+@protocol.commands.add("tagtypes")
+def tagtypes(context, *args):
+    """
+    *mpd.readthedocs.io, connection settings section:*
+
+        ``tagtypes``
+
+        Shows a list of available song metadata.
+
+        ``tagtypes disable {NAME...}``
+
+        Remove one or more tags from the list of tag types the client is interested in.
+
+        ``tagtypes enable {NAME...}``
+
+        Re-enable one or more tags from the list of tag types for this client.
+
+        ``tagtypes clear``
+
+        Clear the list of tag types this client is interested in.
+
+        ``tagtypes all``
+
+        Announce that this client is interested in all tag types.
+    """
+    if not args:
+        pass
+    elif args[0] == "all":
```

Comment:
```suggestion
    elif args[0].lower() == "all":
```
and for the others.

In fact, better doing something like the original version:
```
args = list(args)
    if args:
        subcommand = args.pop(0).lower()
```